opencti-platform/opencti-front/src/private/components/threats/threat_actors_individual/ThreatActorIndividualDemographics.tsx

```diff
@@ -0,0 +1,282 @@
+) => {
+  const classes = useStyles();
+  const { t, fsd } = useFormatter();
+
+  const [countryRelationship, setCountryRelationships] = useState<ThreatActorIndividualDemographicsCountryRelationships>({
+    x_mcas_country_of_residence: [],
+    x_mcas_citizenship: [],
+    x_mcas_place_of_birth: undefined,
+  });
+
+  fetchQuery(
+    getDemographicCountryRelationship,
+    { id: threatActorIndividual?.id },
+  ).subscribe({
+    closed: false,
+    error: () => {},
+    complete: () => {},
+    next: (
+      data: ThreatActorIndividualDemographicsCountryRelationshipsQuery$data,
+    ) => {
+      const fetchedCountryRelationships:
+      ThreatActorIndividualDemographicsCountryRelationships = {
+        x_mcas_country_of_residence: [],
+        x_mcas_citizenship: [],
+        x_mcas_place_of_birth: data?.threatActorIndividual?.bornIn?.name,
+      };
+      const edges = data
+        ?.threatActorIndividual
+        ?.stixCoreRelationships
+        ?.edges
+          ?? [];
+      for (const { node } of edges) {
+        const { relationship_type } = node ?? {};
+        const name = node?.to?.name;
+        if (name) {
+          switch (relationship_type) {
+            case 'resides-in':
+              fetchedCountryRelationships.x_mcas_country_of_residence.push(name);
+              break;
+            case 'citizen-of':
+              fetchedCountryRelationships.x_mcas_citizenship.push(name);
+              break;
+            default:
+          }
+        }
+      }
+      setCountryRelationships(fetchedCountryRelationships);
+    },
+  });
+
+  function toVal(value: string, dict: Record<string, string>) {
+    return t(Object.values(dict)[Object.keys(dict).indexOf(value)]);
+  }
+
+  return (
+    <div style={{ height: '100%' }}>
+      <Typography variant="h4" gutterBottom={true}>
+        {t('Demographic Information')}
+      </Typography>
+      <Paper classes={{ root: classes.paper }} variant="outlined">
+        <Grid container={true} spacing={3}>
+          {/* Row #1 */}
+          <Grid item={true} xs={3} style={{ marginTop: -1 }}>
+            <Typography variant="h3" gutterBottom={true} style={{ float: 'left' }}>
+              {t('Country of Residence')}
+            </Typography>
+          </Grid>
+
+          <Grid item={true} xs={3} style={{ marginTop: -1 }}>
+            <Typography variant="h3" gutterBottom={true} style={{ float: 'left' }}>
+              {t('Citizenship')}
+            </Typography>
+          </Grid>
+
+          <Grid item={true} xs={3} style={{ marginTop: -1 }}>
+            <Typography variant="h3" gutterBottom={true} style={{ float: 'left' }}>
+              {t('Place of Birth')}
+            </Typography>
+          </Grid>
+
+          <Grid item={true} xs={3} style={{ marginTop: -1 }}>
+            <Typography variant="h3" gutterBottom={true} style={{ float: 'left' }}>
+              {t('Date of Birth')}
+            </Typography>
+          </Grid>
+
+          {/* Row #2 */}
```

Comment:
Instead of displaying Demographics by rows, you should display by field type (get inspiration from ThreatActorIndividualDetails and all the entities overview), for consistency and design reasons